HW2/src/main/java/com/example/hw2/entity/Car.java

```diff
@@ -0,0 +1,18 @@
+package com.example.hw2.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "car")
+public class Car {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```

Comment:
Is this entity works well without providing a specific sequence name?

*the same question for next entities